### PR TITLE
Admin-only RPDs with no delay

### PR DESCRIPTION
**Pull request by DamianX**
Mostly for local testing.

demonstrated regular first and admin second:

### CODE REVIEW

**Comment by SysadminJeroen** [comment]
What about buildmode?

**Comment by DamianX** [comment]
Yeah, this is mostly for the RPD

**Comment by SysadminJeroen** [comment]
Not that I oppose this, but it just feel like this is what buildmode can already do.

**Comment by DamianX** [comment]
...buildmode can place pipes?

**Comment by jakmak6** [comment]
Why not just have this be a CE item or an R&D item?

**Comment by Kurfursten** [comment]
@jakmak6 

>Pick up CE RPD
>Turn on autoclicker, set to 1000 clicks per second
>Create lag

**Comment by Intigracy** [comment]
This is a lot easier than buildmode since that requires you to already know the path of the items you want to spawn

**Comment by CptWad** [comment]
Buildmode is good for massive projects but for something like piping this would be a better alternative 
also the title is wrong, you mean admin only **RPD** as the RCD is the worse buildmode

**Comment by DamianX** [comment]
well, TECHNICALLY the RPD is a RCD

**Comment by CptWad** [comment]
\>well, TECHNICALLY the RPD is a RCD
\>well, TECHNICALLY the Rapid Piping Device is a Rapid Construction Device
I mean you are not wrong, but neither do the same thing

**Comment by PJB3005** [comment]
Actually the typepath for the RPD is a child of the RCD.

File: code/modules/RCD/RPD.dm [hunk 1]
```diff
@@ -142,3 +142,16 @@
 	if (selected)
 		return selected.Topic(href, href_list)
 
+/obj/item/device/rcd/rpd/admin
+	name = "experimental Rapid-Piping-Device (RPD)"
+
+/obj/item/device/rcd/rpd/admin/afterattack(var/atom/A, var/mob/user)
+	if(!user.check_rights(R_ADMIN))
+		visible_message("\The [src] disappears into nothing.")
+		user.drop_item(src, force_drop = TRUE)
```
Comment (PJB3005) [inline code/modules/RCD/RPD.dm]: Isn't this done by deleting the object?

Comment (DamianX) [reply code/modules/RCD/RPD.dm]: Yeah, it is, I'll remove it

File: code/modules/RCD/RPD.dm [hunk 2]
```diff
@@ -142,3 +142,15 @@
...
+	name = "experimental Rapid-Piping-Device (RPD)"
+
+/obj/item/device/rcd/rpd/admin/afterattack(var/atom/A, var/mob/user)
+	if(!user.check_rights(R_ADMIN))
+		visible_message("\The [src] disappears into nothing.")
+		qdel(src)
+		return
+	return ..()
+
+/obj/item/device/rcd/rpd/admin/delay(var/amount)
```
Comment (Intigracy) [inline code/modules/RCD/RPD.dm]: Are you positive that putting only the one arg isn't going to make BYOND shit itself?
It's done so in the past.

Comment (DamianX) [reply code/modules/RCD/RPD.dm]: I mean, it works, but let's be sure

**Pull request merged**